.vscode/launch.json

```diff
@@ -25,6 +25,22 @@
       "outFiles": ["${workspaceFolder}/qt-cpp/out/test/**/*.js"],
       "preLaunchTask": "npm qt-cpp watch and test"
     },
+    {
+      "name": "Run Extension Tests qt-cpp build",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}",
+      "args": [
+        "--extensionDevelopmentPath=${workspaceFolder}/qt-cpp",
+        "--extensionTestsPath=${workspaceFolder}/qt-cpp/out/test/suite/index-build",
+        "${workspaceFolder}/qt-cpp/test/tmp-build", 
+        "--disable-workspace-trust",
+        "--disable-restore-windows"
+      ],
+      "outFiles": ["${workspaceFolder}/qt-cpp/out/test/**/*.js"],
+      "preLaunchTask": "qt-cpp: build test prelaunch chain",
```

Comment:
I think we have separated our test code into `test` and `scripts` currently. What we have done in `scripts/clean_test_build.ts` and `scripts/prepare_test_build.ts` can be implemented with a default environment class in our test folder. For example, https://github.com/microsoft/vscode-cmake-tools/blob/main/test/helpers/test/default-environment.ts#L12. The main idea of the `scripts` folder is to manage the project easily with fewer CLI commands, like generating licenses, starting tests, publishing, etc. Having test-specific code is beyond the scope of that folder.